modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/AccountResourceImpl.java

```diff
@@ -182,28 +182,22 @@ public EntityModel getEntityModel(MultivaluedMap multivaluedMap)
 	}
 
 	@NestedField(
-		parentClass = com.liferay.headless.admin.user.dto.v1_0.Organization.class,
-		value = "organizationAccounts"
+		parentClass = Organization.class, value = "organizationAccounts"
 	)
 	@Override
 	public Page<Account> getOrganizationAccountsPage(
 			String organizationId, String search, Filter filter,
 			Pagination pagination, Sort[] sorts)
 		throws Exception {
 
-		Organization organization = _organizationResourceDTOConverter.getObject(
-			organizationId);
-
 		return _getOrganizationAccountsPage(
 			Collections.emptyMap(),
 			booleanQuery -> {
 				BooleanFilter booleanFilter =
 					booleanQuery.getPreBooleanFilter();
 
 				booleanFilter.add(
-					new TermFilter(
-						"organizationIds",
-						String.valueOf(organization.getOrganizationId())),
+					new TermFilter("organizationIds", organizationId),
```

Comment:
Hey @MrJohn1911, please keep using the  `_organizationResourceDTOConverter` since the `organizationId` parameter may not equal the organization primaryKey in all cases.

Thx